fhir-persistence/src/main/java/com/ibm/fhir/persistence/FHIRPersistence.java

```diff
@@ -258,33 +260,35 @@ default boolean isChangesSupported() {
      * @return a list containing up to resourceCount elements describing resources which have changed
      * @throws FHIRPersistenceException
      */
-    List<ResourceChangeLogRecord> changes(int resourceCount, java.time.Instant sinceLastModified,
+    List<ResourceChangeLogRecord> changes(FHIRPersistenceContext context, int resourceCount, java.time.Instant sinceLastModified,
         java.time.Instant beforeLastModified, Long changeIdMarker, List<String> resourceTypeNames,
         boolean excludeTransactionTimeoutWindow, HistorySortOrder historySortOrder)
         throws FHIRPersistenceException;
 
     /**
      * Erases part or a whole of a resource in the data layer.
      *
+     * @param context
```

Comment:
just to be consistent
```suggestion
     * @param context the FHIRPersistenceContext associated with this request
```

Reply:
done